BDArmory/VesselSpawning/ContinuousSpawning.cs

```diff
@@ -124,6 +126,36 @@ private IEnumerator SpawnVesselsContinuouslyCoroutine(CircularSpawnConfig spawnC
                 spawnFailureReason = SpawnFailureReason.NoCraft;
                 yield break;
             }
+            var npc_folder = Path.Combine(Path.Combine(AutoSpawnPath, spawnConfig.folder), "NPCs");
```

Comment:
This can be written as
```csharp
var npc_folder = Path.Combine(AutoSpawnPath, spawnConfig.folder, "NPCs");
```
`Path.Combine` has variants for up to 4 arguments and a variant for an array of paths.